compute_tools/src/bin/compute_ctl.rs

```diff
@@ -111,11 +111,6 @@ fn main() -> Result<()> {
 fn init() -> Result<(String, clap::ArgMatches)> {
     init_tracing_and_logging(DEFAULT_LOG_LEVEL)?;
 
-    opentelemetry::global::set_error_handler(|err| {
```

Comment:
@arpad-m, are you okay with getting OTel errors at ERROR level? Technically, you should set `OTEL_SDK_DISABLED=true` to disable span processing and export.

Reply:
The behaviour before was to print the raw error to stdout without using tracing. Which then lacks stuff like timestamps, ability to filter, etc. As I was investigating a race condition, timestamps were important for me so I removed the noise.

Looking at https://github.com/open-telemetry/opentelemetry-rust/pull/2128/files it seems to use tracing now, so basically does what we want.

The log level is fine by me, but it's also not my team's code, so maybe someone from the compute team should weigh in.